### PR TITLE
fix: view cluster as a non-admin

### PR DESCRIPTION
**Pull request by keita-determined**
## Description

[WEB-707]

## Test Plan

There are 2 patterns of test plans

### IF RBAC is **_OFF_**

- Go to cluster page
- Admin and non-admin can view `overview`, `historical usage`, and `master logs` in clusters 

### IF RBAC is **_ON_**

- As Admin
  - Go to cluster page
  - Only admin can view `overview`, `historical usage`, and `master logs` in clusters 

- As Non-admin
  - Go to cluster page
  - Non-admin can only view `overview` in clusters
  - Check if no error notifications show up
  - Check if stat cards are limited compared to admin

## Commentary (optional)
In [ticket](https://determinedai.atlassian.net/browse/WEB-707), it says `Overview tab only (but dont need tabs component)`, but this makes code more complicated unnecessarily, so i used `Tab` for all cases. 

## Checklist

- [x] Changes have been manually QA'd
- [x] User-facing API changes need the "User-facing API Change" label.
- [x] Release notes should be added as a separate file under `docs/release-notes/`.
  See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.
- [x] Licenses should be included for new code which was copied and/or modified from any external code.

[WEB-707]: https://determinedai.atlassian.net/browse/WEB-707?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: webui/react/src/pages/Clusters.tsx [hunk 1]
```diff
@@ -56,24 +59,33 @@ const Clusters: React.FC = () => {
...
+
+    if (rbacEnabled) {
+      tabs.push(...clustersOverview);
+      if (canAdministrateUsers) {
+        tabs.push(...historicalUsage, ...masterLogs);
+      }
+    } else {
+      // if RBAC is not enabled, show all tabs
+      tabs.push(...clustersOverview, ...historicalUsage, ...masterLogs);
+    }
```
Comment (gt2345) [inline webui/react/src/pages/Clusters.tsx:85]: nit: If you remove the '[]' when defining individual tabs, you don't have to use the spread operator in push

**Review by gt2345 — APPROVED** [review]
Works great

File: webui/react/src/pages/Clusters.tsx (hunk 1 above)
Comment (keita-determined) [reply webui/react/src/pages/Clusters.tsx:85]: i couldn't find `Tab` type from antd. `TabsProps['items']` is `Tab[]` alias. 
i think the current way is safer, but i agree that spread operator isnt cool. 
i'll leave it as it is now.

Comment (keita-determined) [reply webui/react/src/pages/Clusters.tsx:90]: i actually found a way to remove spread operator.

**Pull request merged**